corehq/apps/export/utils.py

```diff
@@ -73,5 +76,9 @@ def get_transform_function(func_name):
 
 
 def get_deid_transform_function(func_name):
-    module = importlib.import_module('couchexport.deid')
-    return getattr(module, func_name)
+    if func_name == DEID_TRANSFORM_FUNCTIONS[DEID_DATE_TRANSFORM]:
+        module = importlib.import_module('couchexport.deid')
+        return getattr(module, func_name)
+    elif func_name == DEID_TRANSFORM_FUNCTIONS[DEID_ID_TRANSFORM]:
+        from corehq.apps.export.models import DeIdMapping
+        return DeIdMapping.get_deid
```

Comment:
This function will return `None` when neither of the if-conditions are met. Is that intentional?

Reply:
`deid_transform` that ultimately gets passed in here is set from the list of `DEID_TRANSFORM_FUNCTIONS` as choices:
https://github.com/dimagi/commcare-hq/blob/1a32a6bc9481f31cb8212f7e207b9d33eae1db23/corehq/apps/export/models/new.py#L278

If what gets passed in here is not in that list of choices, something has gone wrong, but I wouldn't expect that to be possible. The caller would then error on trying to call `None` as a function. Is that a behavior worth writing a test for?